tools/merger/data/ci/ui/job.yml.template

```diff
@@ -37,7 +37,7 @@
 {{{custom_steps}}}
       - run:
           name: test
-          command: cd {{{relative_path}}}/{{{test_section}}} && ./node_modules/.bin/jest --ci --maxWorkers 2
+          command: cd {{{relative_path}}}/{{{test_section}}} && ../../node_modules/.pnpm/jest@27.0.6/node_modules/jest/bin/jest.js --ci --maxWorkers 2 --passWithNoTests
```

Comment:
This is the temporary hack to get around https://github.com/pnpm/pnpm/issues/3582

Reply:
Interesting- the symlink causes issues so we're using pnpm's installed jest directly

Reply:
It's a mystery to me too!